src/Symfony/Component/Form/AbstractType.php

```diff
@@ -64,8 +64,16 @@ public function configureOptions(OptionsResolver $resolver)
     /**
      * {@inheritdoc}
      */
+    public function getName()
+    {
+        return;
```

Comment:
All existing form types should still keep the method (i.e. it remains in AbstractType), but it can be removed from the interface. We already did that in same cases AFAIK: remove a method from an interface but keep it in the implementations.